build.gradle

```diff
@@ -56,6 +56,8 @@ dependencies {
     testImplementation 'io.rest-assured:spring-mock-mvc:3.3.0'
     testIntegrationImplementation "com.github.tomakehurst:wiremock:2.27.2"
     testImplementation 'io.swagger.parser.v3:swagger-parser-v2-converter:2.0.25'
+
+    testImplementation 'org.springframework:spring-test'
```

Comment:
odd one, not sure you should really need this - I thought that `spring-boot-starter-test` is included automatically which then includes this dependency